lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -691,98 +725,220 @@ PXR_NS::SdfValueTypeName ufeTypeToUsd(const std::string& ufeType)
     }
 }
 
-PXR_NS::VtValue vtValueFromString(const std::string& typeName, const std::string& strValue)
+VtValue vtValueFromString(const SdfValueTypeName& typeName, const std::string& strValue)
```

Comment:
The Ufe type name was ambiguous. Changed signature to require a USD type name so we can provide correct conversion to VtValue every time.